conda/cli/main_search.py

```diff
@@ -29,7 +29,7 @@ def execute(args, parser):
     if args.envs:
         with Spinner(
             "Searching environments for %s" % spec,
-            not context.verbosity and not context.quiet,
+            context.verbosity < 2 and not context.quiet,
```

Comment:
The `2` should be a constant to easier understand this, e.g.:

```suggestion
            context.verbosity < logging.INFO and not context.quiet,
```

Reply:
we introduce helpful aliases instead, the aliases produce much more readable code:

```
context.verbose and not context.quiet
```